docs/cli/kops_rolling-update.md

```diff
@@ -5,37 +5,52 @@ Rolling update a cluster.
 ### Synopsis
 
 
-This command updates a kubernetes cluster to match the cloud, and kops specifications. 
+This command updates a Kubernetes cluster to match the cloud, and kops specifications. 
 
-To perform rolling update, you need to update the cloud resources first with "kops update cluster" 
+The Examples below include a series of command for Terraform users.  The workflow includes using Terraform. 
 
-Note: terraform users will need run the following commands all from the same directory "kops update cluster --target=terraform" then "terraform plan" then "terraform apply" prior to running "kops rolling-update cluster" 
+Use export KOPS FEATURE FLAGS="+DrainAndValidateRollingUpdate" to use beta code that drains the nodes and validates the cluster.  New flags for Drain and Validation operations will be shown when the environment variable is set. 
 
-Use export KOPS FEATURE FLAGS="+DrainAndValidateRollingUpdate" to use beta code that drains the nodes and validates the cluster.  New flags for Drain and Validation operations will be shown when the environment variable is set.
+Node Replacement Algorithm Alpa Feature 
+
+We are now including three new algorithms that influence node replacement. All masters and bastions are rolled sequentially before the nodes, and this flag does not influence their replacement.  These algorithms utilize the feature flag mentioned above. 
+
+  1. "asg" - A node is drained then deleted.  The cloud then replaces the node automatically. (default)  
```

Comment:
What's the main difference between pre-create and create? They seem the same to me. Is the difference that one waits for all node groups to be replace while the other does one at a time?

What do you think of something like this:
1. "default" - A node is drained then deleted.  The cloud then replaces the node automatically. 
2. "CreateBeforeDestroy" - All node instance groups are duplicated first; then all old nodes are cordoned.  

As discussed over slack it's also possible to accomplish "CreateBeforeDestroy" without duplicating instance groups by tagging nodes that should be deleted or using a timestamp to determine which nodes are "old"